chatterino.pro

```diff
@@ -406,6 +407,7 @@ HEADERS += \
     src/messages/MessageElement.hpp \
     src/messages/MessageParseArgs.hpp \
     src/messages/search/AuthorPredicate.hpp \
+    src/messages/search/ChannelPredicate.hpp \
```

Comment:
You will also need the new files to the `CMakeLists.txt` in order to support both build systems. (Also resolves CI issue)